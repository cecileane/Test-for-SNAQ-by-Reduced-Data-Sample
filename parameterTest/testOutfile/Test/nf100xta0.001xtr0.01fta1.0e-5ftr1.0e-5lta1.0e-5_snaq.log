optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-5,
                       xtolAbs=0.001, xtolRel=0.01.
 max number of failed proposals = 100, liktolAbs = 1.0e-5.
rootname for files: nf100xta0.001xtr0.01fta1.0e-5ftr1.0e-5lta1.0e-5_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 59038
seed: 59038 for run 1
Sat Oct  1 17:32:59 2016

 BEGIN SNaQ for run 1, seed 59038 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 122 (failures: 100) with -loglik=155.49626 and ht_min=[0.12573,0.33535,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 30		 30		 1	 |	 0		 0		 --
mvtarget	 27		 27		 1	 |	 0		 0		 --
chdir		 8		 8		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 56		 46		 0	 |	 2		 2		 --
Total		 122		 112		 3	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 155.49625734560587
((((5,6):0.33534665830782573,3):0.12573352700249715,(1,(2)#H7:::0.6269753415308497):2.1558604888533637):2.1558604888533637,4,#H7:::0.3730246584691503);
---------------------
seed: 25877 for run 2
Sat Oct  1 17:33:05 2016

 BEGIN SNaQ for run 2, seed 25877 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 136 (failures: 100) with -loglik=126.74251 and ht_min=[0.2856,0.51916,1.51213,10.0,0.48942]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 2	 |	 NA 		 NA 		 NA 
mvorigin	 30		 30		 3	 |	 7		 7		 --
mvtarget	 30		 30		 0	 |	 6		 6		 --
chdir		 10		 10		 1	 |	 14		 14		 --
delete		 0		 0		 0	 |	 1		 1		 --
nni		 64		 49		 0	 |	 2		 2		 --
Total		 136		 121		 6	 |	 30		 30		 6
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 0.2

 FINISHED SNaQ for run 2, -loglik of best 126.74250981974937
(((3,((2,(1)#H9:::0.7143969586544817):1.512133678414306,4):0.4894188376235174):0.519160748086888,5):10.0,6,#H9:::0.2856030413455183);
---------------------

Sat Oct  1 17:33:17 2016
MaxNet is (((3,((2,(1)#H9:::0.7143969586544817):1.512133678414306,4):0.4894188376235174):0.519160748086888,5):10.0,6,#H9:::0.2856030413455183); 
with -loglik 126.74250981974937
