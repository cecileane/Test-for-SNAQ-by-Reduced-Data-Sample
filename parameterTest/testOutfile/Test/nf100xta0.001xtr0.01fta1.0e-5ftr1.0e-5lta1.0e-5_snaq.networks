(((3,((2,(1)#H9:::0.7143969586544817):1.512133678414306,4):0.4894188376235174):0.519160748086888,5):10.0,6,#H9:::0.2856030413455183);, with -loglik 126.74250981974937 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,((2,(1,#H9:::0.3094340755382883):6.8748090159083075):0.6558946413498205,4):0.4532083076293834,(5,(6)#H9:::0.6905659244617117):1.6333078236283967);, with -loglik 128.33114494962908
(((3,(4)#H9:::0.6084400047101367):0.030273491082867903,5):0.25485080305975955,6,(1,(#H9:::0.3915599952898633,2):3.050674566570035):0.7201999246673064);, with -loglik 186.86512683542443
((#H9:::0.40511587410688876,5):4.221035177589643,6,(1,(((3)#H9:::0.5948841258931112,4):0.0633024359440199,2):0.12800961493237392):0.6933687798767576);, with -loglik 223.12929207762653
(((3,(#H9:::0.3023867605233367,4):2.0496033435707077):0.0,5):0.19148277758496798,6,(1,(2)#H9:::0.6976132394766633):0.43290265533805844);, with -loglik 268.0532685409899
((5)#H9:::0.5469165672088636,6,(1,(((3,#H9:::0.4530834327911364):1.0182177932290677,4):0.09895222520290604,2):0.11887836545937482):0.032578884480645415);, with -loglik 383.02893603262333
