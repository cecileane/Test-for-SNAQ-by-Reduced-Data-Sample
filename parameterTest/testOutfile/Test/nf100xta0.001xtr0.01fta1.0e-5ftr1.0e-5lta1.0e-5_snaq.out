(((3,((2,(1)#H9:::0.7143969586544817):1.512133678414306,4):0.4894188376235174):0.519160748086888,5):10.0,6,#H9:::0.2856030413455183); -Ploglik = 126.74250981974937
 Dendroscope: (((3,((2,(1)#H9):1.512133678414306,4):0.4894188376235174):0.519160748086888,5):10.0,6,#H9);
 Elapsed time: 18.511315543 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (((3,((2,(1)#H9:::0.7143969586544817):1.512133678414306,4):0.4894188376235174):0.519160748086888,5):10.0,6,#H9:::0.2856030413455183);, with -loglik 126.74250981974937
 ((((5,6):0.33534665830782573,3):0.12573352700249715,(1,(2)#H7:::0.6269753415308497):2.1558604888533637):2.1558604888533637,4,#H7:::0.3730246584691503);, with -loglik 155.49625734560587
-------
